### PR TITLE
centos-stream targets/chroots & TF's composes

### PR DESCRIPTION
**Pull request by jpopelka**

### CODE REVIEW

**Review by TomasTomecek — APPROVED** [review]
neat!

File: content/docs/testing-farm.md [hunk 1]
```diff
@@ -29,6 +29,9 @@ In order to enable test execution simply include `tests` jobs in the `.packit.ya
 Required metadata:
 * **targets** - You can use `fedora-development`, `fedora-stable`, `fedora-all`
   or release specific targets such as `fedora-34-x86_64`.
+  Also `centos-7`, `centos-8` and `centos-stream-8`. The targets are mapped to
+  Testing Farm's [composes](https://api.dev.testing-farm.io/v0.1/composes)
+  when submitting the tests.
```
Comment (lachmanfrantisek) [inline content/docs/testing-farm.md:34]: Do you think we want to mention internal TF as well? (It has a different mapping.) Interested users can contact us/TFT directly I think.

**Review by lachmanfrantisek — APPROVED** [review]
Thanks!

File: content/docs/testing-farm.md (hunk 1 above)
Comment (jpopelka) [reply content/docs/testing-farm.md:34]: We don't mention internal TF anywhere else in the docs (yet), do we?
So I'll merge this but you can add more details about internal TF later, if you want.

**Pull request merged**